kubernetes/cmsweb/services/reqmon.yaml

```diff
@@ -58,11 +58,11 @@ spec:
         name: reqmon
         resources:
           requests:
-            memory: "256Mi"
-            cpu: "300m"
+            memory: "5Gi"
+            cpu: "1000m"
           limits:
             memory: "8Gi"
-            cpu: "1000m"
+            cpu: "1100m"
```

Comment:
Can reqmon use more than 1 core? If not, there is no need to assign 1100m here, you can up 1000m instead.

Reply:
No, it's meant to run in a single process. I'll set request and limit to the same value then (1000m).

Reply:
BTW, can't we have issues if a pod requires 1000m CPU, but also say that this is the limit?

Reply:
@amaltaro 
No, there is no issue. I have checked this. The `requests` value is used to schedule the service if the resources are available. Once the service is scheduled, it may consume less than the `requests` value. The `limits` are used to make sure the pod never consumes greater than this value. Of course `limit` can never be lower than the `request` but these can be equal.

Reply:
Thanks for the clarification, Imran. I'm going to update this PR